test/controllers/project_controller_test.exs

```diff
@@ -58,43 +37,26 @@ defmodule CodeCorps.ProjectControllerTest do
     end
   end
 
-  describe "#show" do
+  describe "show" do
     test "shows chosen resource", %{conn: conn} do
-      project = insert(:project,
-        title: "Test project",
-        description: "Test project description",
-        long_description_markdown: "A markdown **description**")
+      project = insert(:project)
 
-      conn = get conn, project_path(conn, :show, project)
-      data = json_response(conn, 200)["data"]
-      assert data["id"] == "#{project.id}"
-      assert data["type"] == "project"
-      assert data["attributes"]["title"] == "Test project"
-      assert data["attributes"]["description"] == "Test project description"
-      assert data["attributes"]["long-description-markdown"] == "A markdown **description**"
-      assert data["relationships"]["organization"]["data"]["id"] == Integer.to_string(project.organization_id)
+      conn
+      |> request_show(project)
+      |> json_response(200)
+      |> Map.get("data")
+      |> assert_result_id(project.id)
     end
 
     test "shows chosen resource retrieved by slug", %{conn: conn} do
-      project = insert(:project,
-        title: "Test project",
-        description: "Test project description",
-        long_description_markdown: "A markdown **description**",
-        slug: "test-project")
+      project = insert(:project, title: "Test project", slug: "test-project")
 
       conn = get conn, "/test-organization/test-project"
 
-      data =
         conn
         |> json_response(200)
         |> Map.get("data")
-
-      assert data["id"] == "#{project.id}"
-      assert data["type"] == "project"
-      assert data["attributes"]["title"] == "Test project"
-      assert data["attributes"]["description"] == "Test project description"
-      assert data["attributes"]["long-description-markdown"] == "A markdown **description**"
-      assert data["relationships"]["organization"]["data"]["id"] == Integer.to_string(project.organization_id)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1925540/20103504/8d7fbbe8-a5ca-11e6-98f9-4875a0cd9514.png)

This is what I love the most about these.